enterprise/internal/authz/authz.go

```diff
@@ -252,3 +253,10 @@ func ProviderFromExternalService(siteConfig schema.SiteConfiguration, svc *types
 	}
 	return providers[0], nil
 }
+
+func ParseInterval(interval int) time.Duration {
+	if interval == 0 {
```

Comment:
```suggestion
	if interval <= 0 {
```

nit: just to be very cautious ...